docs/manual/manual_redundant_throttle.rst

```diff
@@ -145,4 +147,28 @@ The example is demonstrated below:
     </style>
     <video class='center_vid' controls><source src="../_static/manual_images/redundant_throttle/redundant_arming_example.mp4" type="video/mp4"></video>
 
-.. _config_values:
\ No newline at end of file
+DroneCAN and PWM Arming interaction
+#######################################
+
+Vertiq modules are configured to disarm on the first received PWM signal. As such, if your module has already armed via DroneCAN, and after some time a PWM throttle is received, 
+the module will disarm even if DroneCAN has a higher priority. The image below illustrates this behavior.
+
+.. image:: ../_static/manual_images/redundant_throttle/pwm_force_disarm.png
+
+DroneCAN and DSHOT Arming interaction

```

Comment:
Not really a thing to change here, but a thing to think about over the long-term as we do UX stuff. Maybe disarming stuff should have some kind of source stuff attached to it eventually to smooth out these kind of weird loopy things